src/current/releases/v24.2.md

```diff
@@ -24,6 +24,8 @@ docs_area: releases
 
 {% include releases/whats-new-intro.md major_version=vers %}
 
+## Feature highlights
```

Comment:
@mdlinville There should be one Feature Highlights heading (within the v24.2.0 section, not here). If this is temporary, to get the build to pass while we worked on including the other "Feature highlights" section, please ignore this and resolve when ready.

Reply:
It was, but thanks for the reminder. I don't need it anymore.